### PR TITLE
UI references in Command Reference

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -1157,3 +1014,162 @@ WANdisco LiveMigrator >> exclusion show --id 100mbfiles
...
+        stacktrace: Display the full stacktrace of the last error.
+
+Exclusion Commands
+        exclusion add file-size: Create a new file size rule.
+        exclusion add regex: Create a new regex exclusion rule.
+        exclusion del: Delete an exclusion rule.
+        exclusion list: List all exclusion rules.
+        exclusion show: Get details for a particular exclusion rule.
+
+Filesystem Commands
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1087]: Line 1087 -> check full stops for consistency.

File: docs/command-reference.md [hunk 2]
```diff
@@ -1157,3 +1014,162 @@ WANdisco LiveMigrator >> exclusion show --id 100mbfiles
...
+#### Optional Parameters
+* **`--command`**, **`-C`** The command for which help information is wanted
+
+#### Examples
+
+```text
+WANdisco LiveMigrator >> help
+AVAILABLE COMMANDS
+
+Built-In Commands
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1071]: Line 1071 -> check fullstops for consistency

File: docs/command-reference.md [hunk 3]
```diff
@@ -1157,3 +1014,162 @@ WANdisco LiveMigrator >> exclusion show --id 100mbfiles
...
+
+OPTIONS
+        --file  file
+                A file to save history to.
+                [Optional, default = <none>]
+```
+
+#### Optional Parameters
+
+* **`--file`** The name of the file in which to save the history of commands
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:1147]: Line 1147: full stop?

File: docs/command-reference.md (hunk 2 above)
Comment (benradstone) [reply docs/command-reference.md:1071]: I purposely decided not to do this now as the page is full of these inconsistencies with full stops.

File: docs/command-reference.md [hunk 4]
```diff
@@ -201,6 +42,7 @@ OPTIONS
 #### Mandatory Parameters
 
 * **`--file-system-id`** The identifier of the source file system resource to delete
+  * UI equivalent = **Storage Name**
```
Comment (shonaisbister-disco) [inline docs/command-reference.md:45]: Line 45 (and subsequent): I'm not sure 'equivalent' is necessary. Can we just have UI: Storage Name (for example)

Comment (benradstone) [reply docs/command-reference.md:45]: I can do that.

**Review by shonaisbister-disco — CHANGES_REQUESTED** [review]
Looks great! Just fix some punctuation in the bulleted lists. I'm also not sure about 'equivalent' and think UI would be OK on its own.

**Pull request merged**